### PR TITLE
Manifest doc fixes

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[ff13cd5](https://github.com/OfficeDev/office-js-docs-pr/commits/ff13cd59d489b49429361d754c332cf9f09d519e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/develop/add-in-manifests.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |
[docs/develop/create-addin-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/develop/create-addin-commands.md) | :white_check_mark:Succeeded | |
[docs/develop/specify-office-hosts-and-api-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/develop/specify-office-hosts-and-api-requirements.md) | :white_check_mark:Succeeded | |
[docs/develop/support-for-task-pane-and-content-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/develop/support-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/update-your-javascript-api-for-office-and-manifest-schema-version.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/develop/update-your-javascript-api-for-office-and-manifest-schema-version.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/includes/file-get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md) | :white_check_mark:Succeeded | |
[docs/project/create-a-project-add-in-that-uses-rest-with-an-on-premises-odata-service.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/project/create-a-project-add-in-that-uses-rest-with-an-on-premises-odata-service.md) | :white_check_mark:Succeeded | |
[docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md) | :white_check_mark:Succeeded | |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | |
[docs/word/dictionary-task-pane-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/word/dictionary-task-pane-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201909262006326909-1331%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=CyooMkeA1izRH3zP9hIoTC4kLxlJzDccwtEKIIHZytg%3D&st=2019-09-26T20%3A02%3A37Z&se=2019-10-27T20%3A07%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/concepts/privacy-and-security.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ The following example shows how a task pane add-in specifies the  **ReadDocument
 ```xml
 <?xml version="1.0" encoding="utf-8"?>
 <OfficeApp xmlns="http://schemas.microsoft.com/office/appforoffice/1.0"
-           xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance" 
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (AlexJerabek) [inline docs/concepts/privacy-and-security.md]: I'm curious why this change was made. I get redirected to the secure URL in a browser.

File: docs/develop/add-in-manifests.md [hunk 2]
```diff
@@ -119,13 +123,15 @@ The following XML manifest example hosts its main add-in page in the `https://ww
 
 ```XML
 <?xml version="1.0" encoding="UTF-8"?>
-<OfficeApp xmlns="http://schemas.microsoft.com/office/appforoffice/1.1" xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance" xsi:type="TaskPaneApp">
+<OfficeApp xmlns="http://schemas.microsoft.com/office/appforoffice/1.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="TaskPaneApp">
+  <!--IMPORTANT! Id must be unique for each add-in. If you copy this manifest ensure that you change this id to your own GUID -->
```
Comment (AlexJerabek) [inline docs/develop/add-in-manifests.md]: This (and the other GUID comments) should end in a period.

Comment (AlexJerabek) [inline docs/develop/add-in-manifests.md]: Consider changing "If you copy this manifest ensure that you change this id to your own GUID" to something more direct. Like "Change this id to the GUID of your add-in." (or however that GUID acquisition should be phrased).

File: docs/concepts/privacy-and-security.md (hunk 1 above)
Comment (Rick-Kirkham) [reply docs/concepts/privacy-and-security.md]: I'm not an XML expert, but I know that XML namespace names aren't necessarily actual URLs. My guess would be that sometime since 2001, the www.w3.org started redirecting http requests to it's domain to https.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b62f648](https://github.com/OfficeDev/office-js-docs-pr/commits/b62f6484f9fb9db49eff4c383fa7cee5a23a902b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/privacy-and-security.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/concepts/privacy-and-security.md) | :white_check_mark:Succeeded | |
[docs/develop/add-in-manifests.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/develop/add-in-manifests.md) | :white_check_mark:Succeeded | |
[docs/develop/create-addin-commands.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/develop/create-addin-commands.md) | :white_check_mark:Succeeded | |
[docs/develop/specify-office-hosts-and-api-requirements.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/develop/specify-office-hosts-and-api-requirements.md) | :white_check_mark:Succeeded | |
[docs/develop/support-for-task-pane-and-content-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/develop/support-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |
[docs/develop/update-your-javascript-api-for-office-and-manifest-schema-version.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/develop/update-your-javascript-api-for-office-and-manifest-schema-version.md) | :white_check_mark:Succeeded | |
[docs/excel/custom-functions-overview.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/excel/custom-functions-overview.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/includes/file-get-the-whole-document-from-an-add-in-for-powerpoint-or-word.md) | :white_check_mark:Succeeded | |
[docs/project/create-a-project-add-in-that-uses-rest-with-an-on-premises-odata-service.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/project/create-a-project-add-in-that-uses-rest-with-an-on-premises-odata-service.md) | :white_check_mark:Succeeded | |
[docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/project/create-your-first-task-pane-add-in-for-project-by-using-a-text-editor.md) | :white_check_mark:Succeeded | |
[docs/project/project-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/project/project-add-ins.md) | :white_check_mark:Succeeded | |
[docs/word/dictionary-task-pane-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/manifest-doc-fixes/docs/word/dictionary-task-pane-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C9%5C26%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201909262150105540-1331%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=6MbmTuGBzEG3EPLrK7vUg1DPwmtSWuKzxVQxPl4ZPKE%3D&st=2019-09-26T21%3A46%3A17Z&se=2019-10-27T21%3A51%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.
